src/collect.js

```diff
@@ -96,7 +96,9 @@ function getFlowBin() {
   return process.env.FLOW_BIN || flowBin;
 }
 
-function executeFlow(stdin, root, filepath) {
+let didExecute = false;
+
+function executeFlow(stdin, root, stopOnExit, filepath) {
```

Comment:
Let's annotate the types of function params and all other newly created functions added in this PR